src/sdk/namespace_cache.js

```diff
@@ -739,6 +741,35 @@ class NamespaceCache {
         return this.namespace_hub.get_blob_block_lists(params, object_sdk);
     }
 
+    //////////
+    // ACLs //
+    //////////
+
+    async get_object_acl(params, object_sdk) {
+        if (config.NAMESPACE_CACHING.ACL_HANDLING === "reject") {
+            throw new S3Error(S3Error.AccessDenied);
+        }
+
+        if (config.NAMESPACE_CACHING.ACL_HANDLING === "pass-through") {
+            return this.namespace_hub.get_object_acl(params, object_sdk);
+        }
+
+        return {
+            owner: s3_utils.DEFAULT_S3_USER,
+            access_control_list: [{
+                Grantee: { ID: s3_utils.DEFAULT_S3_USER, Type: "CanonicalUser" },
+                Permission: "FULL_CONTROL"
+            }]
+        };
+    }
+
+    async put_object_acl(params, object_sdk) {
+        if (config.NAMESPACE_CACHING.ACL_HANDLING === "reject") {
+            throw new S3Error(S3Error.AccessDenied);
+        }
+
+        return this.namespace_hub.put_object_acl(params, object_sdk);
```

Comment:
Should we call the hub only when ACL_HANDLING is pass-through?
When ACL_HANDLING is empty string we can do the same as namespace_nb does and call read_object_md and then return without applying the ACL.

Reply:
Should we return nothing, then?

Reply:
Yes